manager/integration/tests/test_basic.py

```diff
@@ -4438,3 +4439,122 @@ def test_space_usage_for_rebuilding_only_volume():  # NOQA
 
     """
     pass
+
+
+def backup_failed_cleanup(client, core_api, volume_name, volume_size,  # NOQA
```

Comment:
nit: would suggest having volume_size fixed to ensure the following status check is always correct.